libkbfs/data_types.go

```diff
@@ -94,7 +94,7 @@ type EncryptedTLFCryptKeyClientAndEphemeral struct {
 }
 
 const (
-	defaultClientMetadataVer kbfsmd.MetadataVer = kbfsmd.SegregatedKeyBundlesVer
+	defaultClientMetadataVer kbfsmd.MetadataVer = kbfsmd.ImplicitTeamsVer
```

Comment:
This looked excitingly scary at first, but it seems this is only used in tests. Perhaps we should rename it to have a `ForTest` suffix?

Reply:
You're talking about `defaultClientMetadataVer`?  In practice yeah it's used just in tests, but it's actually the default in `ConfigLocal`, if no other code calls `config.SetMetadataVersion()` after construction.  It happens that the libkbfs init code always calls it, but I'm not sure that all other users would.

Reply:
I see; LGTM then!